addons/account/tests/test_early_payment_discount.py

```diff
@@ -88,6 +153,39 @@ def test_register_discounted_payment_on_single_invoice(self):
             {'amount_currency': 900.0},
         ])
 
+    def test_register_discounted_payment_on_single_invoice_with_fixed_tax(self):
+        self.company_data['company'].early_pay_discount_computation = 'included'
+        fixed_tax = self.env['account.tax'].create({
+            'name': 'Test 0.05',
+            'type_tax_use': 'sale',
+            'amount_type': 'fixed',
+            'amount': 0.05,
+            'price_include': False,
+            'include_base_amount': False,
+        })
+
+        inv_1500_10_percents_discount_tax_incl_15_percents_tax = self.env['account.move'].create({
```

Comment:
```suggestion
        inv = self.env['account.move'].create({
```
You only have one invoice in this test after all^^

+ Same comments as the other test